### ISSUE TITLE
Chip Shortage: TLE8888 options

### ISSUE THREAD

**Issue by rusefillc** [rusefi/hw_microRusEfi#212]
https://octopart.com/search?q=TLE8888

![image](https://user-images.githubusercontent.com/48498823/144659444-f1b678ba-bdc8-4fb4-b8a1-d552b65d7d70.png)

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/147376710-0ce758fe-685c-40ac-a80a-bad5200dd252.png)

**Comment by rusefillc** [comment]
No changes

![image](https://user-images.githubusercontent.com/48498823/150437894-b28ef06e-cd74-4274-873f-5b8a49d9e333.png)

**Comment by rusefillc** [comment]
@JRDMcLAREN are you still getting TLE8888 anywhere or do you just have a lot remaining?

**Comment by JRDMcLAREN** [comment]
I have last 5 pcs. I'm bought "last" 18 pcs in January 2022 from Digikey
..and have last 3 pcs F407VGT6 ....

**Comment by rusefillc** [comment]
@JRDMcLAREN moused has stock 

![image](https://user-images.githubusercontent.com/48498823/206930216-bcfe8ac8-a19b-4da4-b667-4f16f3573592.png)

**Comment by JRDMcLAREN** [comment]
THX, 5pcs ordered ..
and what kind of MCU can be used in near future ... F427VGT7 .. maybe ..
![Screenshot_20221212_195508](https://user-images.githubusercontent.com/94193511/207134040-d7c45087-a7fc-4736-97f9-50beb001ec89.png)

**Comment by rusefillc** [comment]
427vg is already used by MRE

**Comment by JRDMcLAREN** [comment]
TLE8888QKXUMA1 is marked as "End Of Life" on Mouser
![Screenshot_20230119_225958](https://user-images.githubusercontent.com/94193511/213571683-799f21d5-41c2-437d-8dc2-6b125a65efd3.png)

**Comment by rusefillc** [comment]
it's a bit more complicated - primary version is active

![image](https://user-images.githubusercontent.com/48498823/213601257-0cb9a1f0-86ac-4484-9353-0c585c243832.png)

**Comment by JRDMcLAREN** [comment]
email form Mouser 26/01/2023
![Screenshot_20230126_195332](https://user-images.githubusercontent.com/94193511/216817104-ad6ef224-29c2-4f78-9da2-3654c2774356.png)

**Comment by rusefillc** [comment]
@JRDMcLAREN please clarify what are you saying.

@JRDMcLAREN are you saying that mouser info is more reliable than https://www.infineon.com/cms/en/product/power/automotive-conventional-powertrain-ics/engine-management-ic/tle8888qk/ official status which shows "active" for me today?

**Comment by JRDMcLAREN** [comment]
My experience is, when Mouser has some product marked as EoL, then it is END OF LIFE.
..we need to make "new MRE" with any other chip, maybe ....

YES, Infineon says ACTIVE, but datasheet date is 05/2017, Product Brief is 08/2015 and Footprint is 02/2019

..just try to think about it...

(I have last 10pcs on stock)

**Comment by mi-hol** [comment]
> when Mouser has some product marked as EoL, then it is END OF LIFE.

I fully agree with this statement. This would mean the end for producing new MRE boards with current design & components.
What options for replacing TLE8888 do we have?

**Comment by rusefillc** [comment]
I've just ordered enough chips that I can worry about it all later

![image](https://user-images.githubusercontent.com/48498823/219872091-ccc0bdb6-aab4-4f41-8624-a21440c1a453.png)

**Comment by mi-hol** [comment]
Or is maybe [EVAL-L9779WD-SPI](https://www.st.com/resource/en/data_brief/eval-l9779wd-spi.pdf) Evaluation board for L9779WD-SPI an alternative for the future?

Features
- Voltage min/max: 4.15V to 28V.
- No. channels:
  * 4 injector driver
  * 1 canister valve driver
  * 2 Lambda sensor heater
  * 1 Main relay driver
  * 6 relay driver
  * 1 LED driver
  * 4 Ignition driver
- A programmable power stage configurable as: 
double H-bridge to drive a stepper motor driver,
or 
as independent 4 High side and 4 Low side drivers.
- Device registers setting and the full diagnostic are available through SPI.
- Communication protocols: SPI, CAN, (both with dedicated connectors), Kline.
- Access to all relevant pins by test points. Input signal connector compatible with the SPC56MDiscovery
- Possibility to **connect a generic microcontroller boards by using a simple adapter**.

**Comment by rusefillc** [comment]
Let's move this to https://rusefi.com/forum/ where that amazing chip is already discussed in detail

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/229264143-226734ca-fa5d-44fe-8065-dfb244680747.png)

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/229264234-af3971e7-062a-4e36-8a9f-3485dccc969f.png)

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/232887568-9320e0f0-2c8a-4f34-8102-0ea589c609aa.png)
